### PR TITLE
Fixes missing Weh emote on species

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR

Adds weh emote sound on Felinid, Vulpkanin and Harpy.
Also add honk emote sound to Harpy, this is related to the honk reagent or whatever.
Also removes old no longer used Mothroach emotesounds from Nyano days since Mothroach was upstreamed a while ago.

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- fix: Felinids, Harpies and Vulpkanin should now Weh accordingly.

### CODE REVIEW

**Comment by deltanedas** [comment]
real question is how come lizards cant weh

**Comment by FluffiestFloof** [comment]
> real question is how come lizards cant weh

upstream issue :trollface:

**Comment by Piras314** [comment]
weh!

File: Resources/Prototypes/Nyanotrasen/Voice/speech_emote_sounds.yml [hunk 1]
```diff
@@ -56,13 +74,3 @@
...
-    Sigh:
-      collection: FemaleSigh
-
-# mobs
-- type: emoteSounds
-  id: Mothroach
-  sound:
-    path: /Audio/Voice/Moth/moth_squeak.ogg
-    params:
-      variation: 0.125
```
Comment (GreaseMonk) [inline Resources/Prototypes/Nyanotrasen/Voice/speech_emote_sounds.yml:68]: Was removing this intentional?

Comment (FluffiestFloof) [reply Resources/Prototypes/Nyanotrasen/Voice/speech_emote_sounds.yml:68]: Yes

**Pull request merged**